mlflow/go/pkg/query/parser/validate.go

```diff
@@ -90,7 +92,7 @@ func parseValidIdentifier(identifier string) (ValidIdentifier, error) {
 	case datasetIdentifier, "datasets":
 		return Dataset, nil
 	default:
-		return -1, fmt.Errorf("invalid identifier: %s", identifier)
+		return -1, fmt.Errorf("invalid identifier \"%s\": %w", identifier, ErrValidation)
```

Comment:
use `%q` if you want to quote the string - also worth checking other places where we'd want to quote for consistency